hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java

```diff
@@ -28,19 +26,20 @@
  */
 public abstract class HdfsNode implements IOfferEvaluator, ILauncher {
   private final Log log = LogFactory.getLog(HdfsNode.class);
-  private final LiveState liveState;
   private final ResourceFactory resourceFactory;
 
   protected final HdfsFrameworkConfig config;
-  protected final IPersistentStateStore persistenceStore;
+  protected final HdfsState state;
```

Comment:
I'd like to talk about this in a hangout or something to understand.  As it is HdfsState does have it's persistence layer injected.  So depending on the dependency injection module it will use Zookeeper in the real world, or an InMemory implementation in test environments.